src/tools/miri/tests/fail/alloc/global_system_mixup.stderr

```diff
@@ -1,13 +1,13 @@
 error: Undefined Behavior: deallocating ALLOC, which is Rust heap memory, using PLATFORM heap deallocation operation
-  --> RUSTLIB/std/src/sys/pal/PLATFORM/alloc.rs:LL:CC
+  --> RUSTLIB/std/src/sys/alloc/unix.rs:LL:CC
```

Comment:
This won't work, the test is also run on Windows targets.

The test will need a `normalize-stderr-test` that normalizes `alloc/$OS` to always produce the same output.